ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -76,7 +76,8 @@ function convertToValid(key, model) {
     key.lon = Math.min(180, key.lon)
   }
   if (key.radius !== undefined) {
-    key.radius = Math.max(minimumBuffer, key.radius)
+    let tempRadius = Math.max(minimumBuffer, key.radius)
+    key.radius = isNaN(tempRadius) ? model.get('radius') : tempRadius
```

Comment:
:pencil2: The logic here seems strange to me, but this is equivalent:
```suggestion
    if (isNaN(key.radius)) {
        key.radius = model.get('radius')
    }
```

Reply:
hm, not exactly... that says if radius is a number don't enforce the min value.

But the original seems a little funny too, we check if it is a number after we do the max check... according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max if key.radius is NaN, tempRadius will also be NaN

Reply:
are we going to hold the pr up for this?